runtime/Makefile

```diff
@@ -42,6 +42,12 @@ BINARY_NAME=sledgert
 
 # Debugging Flags
 
+# Enables logs of WASI syscalls
+# CFLAGS += -DLOG_WASI
+
+# Writes sandbox stderr to host stderr
```

Comment:
Where is stderr written to if the flag is switched off? Maybe include in the comment.

Reply:
Good idea!